src/lib/TranscriptEditor/TimedTextEditor/Word.js

```diff
@@ -2,29 +2,37 @@ import React, { PureComponent } from 'react';
 import PropTypes from 'prop-types';
 
 class Word extends PureComponent {
-  render() {
-    const data = this.props.entityKey
-      ? this.props.contentState.getEntity(this.props.entityKey).getData()
-      : {};
-
-    const confidence = data.confidence > 0.6 ? 'high' : 'low';
+  generateConfidence = (data) => {
+    return data.confidence > 0.6 ? 'high' : 'low';
```

Comment:
There's an edge case, when the confidence score it's not provided. we should think about how to handle that. coz with the current check, everything is then mark as low confidence, which is not very helpful. 

<img width="539" alt="screen shot 2018-12-17 at 17 21 08" src="https://user-images.githubusercontent.com/4661975/50100212-300edc00-0220-11e9-9def-91da9338a8ab.png">

But yeah this could be a separate ticket.